src/main/java/li/cil/manual/api/detail/ManualDefinition.java

```diff
@@ -138,4 +150,20 @@
      * @param path the path to navigate to.
      */
     void navigate(String path);
+
+    void setDefaultPage(final String defaultPage);
```

Comment:
Shouldn't these be documented as well?

Reply:
yes, good point. IIRC they were the functions that used to be static on the `Impl` class